aspnetcore/mvc/views/tag-helpers/builtin/resources/ImageTagHelper.md

```diff
@@ -0,0 +1,46 @@
+﻿[Back To Built In Tag Helpers List](../../builtin.md)
+
+
+# ImageTagHelper
+
+By [Peter Kellner](http://peterkellner.net) 
+
+
+The `ImageTagHelper` enhances the html `img` (```<img ... ></img>```) tag. It requires a `src` tag be present as well as the `boolean` attribute `asp-append-version` be included.
+
+If the image source (```src```) is a static file on the host web server, a unique cache bursting string is appended as a parameter to the image source.  This insures that if the file on the host web server changes, a unique request url will be generated that includes the updated request parameter. The cache bursting string is a unique value representing the `Sha512` value of the static image file.
```

Comment:
confused on SHA512.  See if all over the place in abbreviations.  Actually, it's SHA256 so changed that at least. @GuardRex (Do I need to mention you since it's your comment?)